@bwrsandman ,Thanks for yur pr, I tested the usage on my local machine but failed, This is error log:
````
1> Command line: "C:\WINDOWS\system32\cmd.exe" /c "%SYSTEMROOT%\System32\chcp.com 65001 >NUL && "C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2019\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\CMAKE\CMake\bin\cmake.exe"  -G "Visual Studio 16 2019" -A Win32  -DCMAKE_CONFIGURATION_TYPES:STRING="Debug" -DCMAKE_INSTALL_PREFIX:PATH="C:\Users\vzhli17\source\repos\test-lib\out\install\x86-Debug" -DCMAKE_TOOLCHAIN_FILE=F:/Feature-test/openxr/vcpkg/scripts/buildsystems/vcpkg.cmake "C:\Users\vzhli17\source\repos\test-lib" 2>&1"
1> Working directory: C:\Users\vzhli17\source\repos\test-lib\out\build\x86-Debug
1> [CMake] -- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19042.
1> [CMake] -- The C compiler identification is MSVC 19.29.30137.0
1> [CMake] -- The CXX compiler identification is MSVC 19.29.30137.0
1> [CMake] -- Detecting C compiler ABI info
1> [CMake] -- Detecting C compiler ABI info - done
1> [CMake] -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe - skipped
1> [CMake] -- Detecting C compile features
1> [CMake] -- Detecting C compile features - done
1> [CMake] -- Detecting CXX compiler ABI info
1> [CMake] -- Detecting CXX compiler ABI info - done
1> [CMake] -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe - skipped
1> [CMake] -- Detecting CXX compile features
1> [CMake] -- Detecting CXX compile features - done
1> [CMake] CMake Error at F:/Feature-test/openxr/vcpkg/scripts/buildsystems/vcpkg.cmake:816 (_find_package):
1> [CMake]   Could not find a package configuration file provided by "openxr_loader"
1> [CMake]   with any of the following names:
1> [CMake] 
1> [CMake]     openxr_loaderConfig.cmake
1> [CMake]     openxr_loader-config.cmake
1> [CMake] 
1> [CMake]   Add the installation prefix of "openxr_loader" to CMAKE_PREFIX_PATH or set
1> [CMake]   "openxr_loader_DIR" to a directory containing one of the above files.  If
1> [CMake]   "openxr_loader" provides a separate development package or SDK, be sure it
1> [CMake]   has been installed.
````
CMakeLists:
````
# CMakeList.txt : CMake project for test-lib, include source and define
# project specific logic here.
#
cmake_minimum_required (VERSION 3.8)

project(test)
# Add source to this project's executable.
add_executable (test-lib "test-lib.cpp" "test-lib.h")

# TODO: Add tests and install targets if needed.
find_package(openxr_loader CONFIG REQUIRED)

target_link_libraries(test-lib PRIVATE OpenXR::headers OpenXR::openxr_loader)
````

--- a/ports/openxr-loader/portfile.cmake
+++ b/ports/openxr-loader/portfile.cmake
@@ -69,9 +69,9 @@ foreach(HEADER ${HEADER_LIST})
 endforeach()
 
 if(VCPKG_TARGET_IS_WINDOWS)
-    vcpkg_cmake_config_fixup(CONFIG_PATH cmake)
-else(VCPKG_TARGET_IS_WINDOWS)
-    vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/openxr)
+    vcpkg_cmake_config_fixup(PACKAGE_NAME OpenXR CONFIG_PATH cmake)
+else()
+    vcpkg_cmake_config_fixup(PACKAGE_NAME OpenXR CONFIG_PATH lib/cmake/openxr)
 endif()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")

--- a/ports/openxr-loader/vcpkg.json
+++ b/ports/openxr-loader/vcpkg.json
@@ -1,10 +1,11 @@
 {
   "name": "openxr-loader",
   "version": "1.0.22",
-  "port-version": 1,
+  "port-version": 2,
   "description": "A royalty-free, open standard that provides high-performance access to Augmented Reality (AR) and Virtual Reality (VR)—collectively known as XR—platforms and devices",
   "homepage": "https://github.com/KhronosGroup/OpenXR-SDK",
-  "supports": "!(arm | uwp)",
+  "license": "Apache-2.0",
+  "supports": "!uwp",
   "dependencies": [
     "jsoncpp",
     {

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5170,7 +5170,7 @@
     },
     "openxr-loader": {
       "baseline": "1.0.22",
-      "port-version": 1
+      "port-version": 2
     },
     "optimus-cpp": {
       "baseline": "0.3.0",

--- a/versions/o-/openxr-loader.json
+++ b/versions/o-/openxr-loader.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "27baef30397d0db0e6cc01d6f60a86e2ff4df238",
+      "version": "1.0.22",
+      "port-version": 2
+    },
+    {
       "git-tree": "9d673fe500c4b38f0806bd4b3c23a82bba897967",
       "version": "1.0.22",
       "port-version": 1

